workspace-utils/src/lib.rs

```diff
@@ -0,0 +1,148 @@
+use crate::transactions::{CallTransaction, ViewTransaction};
+use aurora_workspace_types::AccountId;
+use workspaces::network::NetworkClient;
+use workspaces::types::{KeyType, SecretKey};
+use workspaces::{Account, Worker};
+
+pub mod macros;
+pub mod results;
+pub mod transactions;
+
+#[derive(Debug, Clone)]
+pub enum AccountKind {
+    Account {
+        contract_id: AccountId,
+        inner: workspaces::Account,
```

Comment:
Redundant path to the struct because of already added use on line 5.
```suggestion
        inner: Account,
```